content/source/docs/enterprise/sentinel/index.html.md

```diff
@@ -23,13 +23,22 @@ policy using Terraform itself.
 
 Using Sentinel with Terraform Enterprise involves:
 
-- [Defining the Policies](./import/index.html) - Policies are defined using the
+- [Defining the policies](./import/index.html) - Policies are defined using the
 [policy language](https://docs.hashicorp.com/sentinel/concepts/language) with
 imports for parsing the Terraform plan, state and configuration.
 - [Managing the policies for organizations](./manage-policies.html) - Organization
 owners add policies to their organization by setting the policy name, policy file,
 and the enforcement level. They then group these policies into policy sets to define
 which workspaces the policies are checked against during runs.
+- [Managing policies with VCS](./integrate-vcs.html) - It's encouraged that
+  organizations manage their policies with VCS after getting a basic
+  understanding of how policy management works, which will assist with the
+  deployment of policies at scale.
```

Comment:
This paragraph sounds a little bit over-impersonal. Maybe "After getting an understanding of policy management, organizations can manage..."

Or @nfagerlund might have a suggestion.

Reply:
Re-worded as per your recommendations!